openssh/test/controls/service_works.rb

```diff
@@ -0,0 +1,10 @@
+pkg_ident = `grep pkg_ident results/last_build.env | cut -d \= -f2`.strip
+installation_directory = "/hab/pkgs/#{pkg_ident}"
+command_test = "#{installation_directory}/sbin/sshd  --help 2>&1"
+
+control 'ensure openssh binary service works' do
```

Comment:
Should this maybe be `'ensure the sshd binary runs'` ?

We should definitely make clear the test is only for the binary; and I think generally we should avoid describing that something "works" as it's quite unspecific and might read like we're giving assurance that the entirety of the functionality works.